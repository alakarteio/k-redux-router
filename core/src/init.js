export default () => ({ type: '@@router/INIT' })
