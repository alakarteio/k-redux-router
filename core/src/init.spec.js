/* eslint-env jest */
import init from './init'

describe('init', () => {
  it('should create an init action', () => {
    expect(init()).toMatchSnapshot()
  })
})
