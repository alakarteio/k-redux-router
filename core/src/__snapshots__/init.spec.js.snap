// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should create an init action 1`] = `
Object {
  "type": "@@router/INIT",
}
`;
